### ISSUE TITLE
Can not load "webpack"! TypeError: undefined is not a function

### ISSUE THREAD

**Issue by claudiopro** [rackt/react-router#1855]
When running `npm test`:

```
WARN [preprocess]: Can not load "webpack"!
  TypeError: undefined is not a function
    at Plugin.notifyKarmaAboutChanges (/Users/delphine/Development/Git/rackt/react-router/node_modules/karma-webpack/index.js:108:15)
    at Plugin.<anonymous> (/Users/delphine/Development/Git/rackt/react-router/node_modules/karma-webpack/index.js:72:9)
    at Compiler.applyPlugins (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/node_modules/tapable/lib/Tapable.js:26:37)
    at Watching._done (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/lib/Compiler.js:78:17)
    at Watching.<anonymous> (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/lib/Compiler.js:61:18)
    at Compiler.emitRecords (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/lib/Compiler.js:282:37)
    at Watching.<anonymous> (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/lib/Compiler.js:58:19)
    at /Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/lib/Compiler.js:275:11
    at Compiler.applyPluginsAsync (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Compiler.afterEmit (/Users/delphine/Development/Git/rackt/react-router/node_modules/webpack/lib/Compiler.js:272:8)
```

This is the same as https://github.com/webpack/karma-webpack/issues/70. I'm preparing a PR to upgrade `karma` to 0.13.x.

**Comment by ryanflorence** [comment]
merged #1856

**Comment by claudiopro** [comment]
Sweet, thanks @ryanflorence
